tests/integration/helm_fluentbit_containerd_multiline_logs_test.go

```diff
@@ -0,0 +1,318 @@
+package integration
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"testing"
+	"time"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	log "k8s.io/klog/v2"
+	"sigs.k8s.io/e2e-framework/klient/k8s/resources"
+	"sigs.k8s.io/e2e-framework/klient/wait"
+	"sigs.k8s.io/e2e-framework/klient/wait/conditions"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal"
+	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal/ctxopts"
+	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal/receivermock"
+	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal/stepfuncs"
+	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal/strings"
+)
+
+func Test_Helm_FluentBit_Containerd_Multiline_Logs(t *testing.T) {
+	const (
+		tickDuration           = 3 * time.Second
+		waitDuration           = 3 * time.Minute
+		logRecords             = 4
+		logLoops               = 500
+		multilineLogCount uint = logRecords * logLoops
+	)
+	var (
+		expectedMetrics = internal.DefaultExpectedMetrics
+	)
+
+	// TODO:
+	// Refactor this: we should find a way to inject this into step func helpers
+	// like stepfuncs.WaitUntilPodsAvailable() instead of relying on an implementation
+	// detail.
```

Comment:
There's a PR that tries to address this: https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2012 

Do you mind waiting with this until it gets merged :) ?

Reply:
I think that this PR can wait until #2012 is merged

Reply:
@kkujawa-sumo https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2012 is merged 🎊

Reply:
@pmalek-sumo please take another look